coral-component-popover/src/scripts/Popover.js

```diff
@@ -294,6 +294,8 @@ class Popover extends Overlay {
     if (target && target.tagName === 'CORAL-COACHMARK') {
       this.setAttribute('variant', variant._COACHMARK);
     }
+
+    this.open = this.open;
```

Comment:
Couldn't get this change?

Reply:
Executes setter which sets `aria-expanded` on new target if it has `aria-haspopup`.